pkg/arvo/app/chat-store.hoon

```diff
@@ -240,12 +242,12 @@
     letter(output (eval bol hoon))
   letter
 ::
-++  append-envelope
+++  prepend-envelope
```

Comment:
idk, it's still (semantically) appending to the message log, even if that (technically) means prepending to the list.

Reply:
That's very confusing and I want to avoid weird naming like that.